### PR TITLE
fix: typo spelling grammar

### PR DESCRIPTION
**Pull request by slowy07**
*fix typo spelling grammar, and repalce to correct word with reference from [merriam-webster](merriam-webster.com) and [wikitionary](wikitionary.org)*

## issue link
*not included because minor fixing which fixing typo*

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: dev/tools/localization/language_subtag_registry.dart [hunk 1]
```diff
@@ -46629,7 +46629,7 @@ Prefix: de
 Type: variant
 Subtag: abl1943
 Description: Orthographic formulation of 1943 - Official in Brazil
-  (Formulário Ortográfico de 1943 - Oficial no Brasil)
```
Comment (goderbauer) [inline dev/tools/localization/language_subtag_registry.dart:46632]: While no native speaker of the language, I believe "Oficial" was correct here. Can you revert this?

File: packages/flutter_tools/lib/src/localizations/language_subtag_registry.dart [hunk 2]
```diff
@@ -46629,7 +46629,7 @@ Prefix: de
 Type: variant
 Subtag: abl1943
 Description: Orthographic formulation of 1943 - Official in Brazil
-  (Formulário Ortográfico de 1943 - Oficial no Brasil)
```
Comment (goderbauer) [inline packages/flutter_tools/lib/src/localizations/language_subtag_registry.dart:46632]: same.

Comment (slowy07) [reply packages/flutter_tools/lib/src/localizations/language_subtag_registry.dart]: ok sir, changes on commit [205d575](https://github.com/flutter/flutter/pull/88439/commits/205d575a38a3b6eb2ad4cb3f9df5e07d54d05875)

File: dev/tools/localization/language_subtag_registry.dart (hunk 1 above)
Comment (slowy07) [reply dev/tools/localization/language_subtag_registry.dart]: done sir !

**Review by HansMuller — APPROVED** [review]
LGTM

You'll need to ask @Hixie for a "docs only" no-tests exception

**Comment by slowy07** [comment]
okay sir @HansMuller, thanks for the reviews !

**Comment by Hixie** [comment]
test-exempt: only affects comments

**Pull request merged**